### DISCUSSION TITLE
How do I turn off having to put a URL for each part?

### DISCUSSION THREAD

**Discussion by InvMgmtSys** [inventree/InvenTree#8041] [Q&A]
### Deployment Method

- [ ] Installer
- [ ] Docker Development
- [ ] Docker Production
- [ ] Bare metal Development
- [ ] Bare metal Production
- [X] Digital Ocean image
- [ ] Other (please provide a link `Steps to Reproduce`

### Describe the problem*

I finished the install using the digitalocean droplet set up. Went to add my first part. I keep getting prompted to add an external URL on each item I try to add. I can't seem to find an option to turn it off. I came across a thread on Reddit where a user reported the same thi g, and he resolved it by doing a reinstall. I would like to avoid that if possible. Any thoughs?

### Steps to Reproduce

Open the new part form
Fill in the category, part name, etc
Do not fill in the url field toward the bottom 
Click save

### Relevant log output

```bash
Na
```

**Comment by matmair** [comment]
Issues are not for questions, please use discussions in the future

**Comment by SchrodingersGat** [comment]
It looks like this is the related discussion - https://github.com/inventree/InvenTree/discussions/7675

No idea why that occured for the other user, or here. Can you please share the exact error message / screenshot you are seeing?
